messages/run.js

```diff
@@ -26,7 +26,9 @@ module.exports = {
 		"tsconfigDescription": "Location of tsconfig.json file",
 		"tsconfigDescriptionLong": "Location of tsconfig.json file used by eslint-typescript engine.",
 		"vceDescription": "Throws an error when violations are detected",
-		"vceDescriptionLong": "Throws an error when violations are detected. Exit code is the most severe violation."
+		"vceDescriptionLong": "Throws an error when violations are detected. Exit code is the most severe violation.",
+		'engineDescription': "engine(s) to run",
```

Comment:
Should the 'e' in 'engine(s)' be uppercase?

Reply:
This follows the convention of categories. From what I can tell, the CLI prefers lowercase in general.

Reply:
@jfeingold35 the it->describe changes are necessary for the tests to pass.

Reply:
In that case, you should also change `tsconfigDescription` and `vceDescription` to follow the convention.

Reply:
@jfeingold35 i can make this consistent, can you approve the review, instead of using the `Request Change`? I would like to submit as soon as I make the change instead of waiting for another cycle.